cfconfiguration/cffinder_unix_test.go

```diff
@@ -0,0 +1,91 @@
+// +build !windows
+
+package cfconfiguration
+
+import (
+	"errors"
+	"github.com/stretchr/testify/assert"
+	"os"
+	"os/user"
+	"testing"
+)
+
+func TestMain(m *testing.M) {
+	retCode := m.Run()
+	os.Exit(retCode)
+}
+
+func TestGetUserConfigFile(t *testing.T) {
+	t.Run("File exist", func(t *testing.T) {
+		tmp := checkFileExistence
```

Comment:
You are repeating a lot of code in each test. You could extract it to separate function.

Reply:
Got rid of patching routines, now code is cleaner.